views/event_details.hbs

```diff
@@ -19,10 +19,25 @@
     </h4>
   {{/if}}
 
-  <div class="margins">
-    <h3 class="dark-text">
+  <div>
+    <h1 class="dark-text" style="font-size: 1.8rem; display: inline-block; margin-right: 1rem;">Place: {{placeDetails.name}}</h1>
+    {{#if website}}
+      {{#if facebook}}
+          <a href={{placeContact.website}}><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-facebook-square fa-2x"></i></a>
+      {{else}}
+        <a href={{placeContact.website}}><i style="margin-bottom: 1rem; color: #409a93;" class="fa fa-link fa-2x"></i></a>
```

Comment:
Add text to indicate it's a link to the website and to make it more clickable

Reply:
I feel like the icon should be self explanatory no need to add text to make it clearer.

Reply:
I disagree and I'm pretty sure @matthewdking did too :upside_down_face: For Facebook the logo is definitely enough, but I think for a website it should at least say something like "visit our website" or just "website" along with the link icon.

Reply:
Merge conflicts and also this.

Reply:
don't you think that's a bit weird though ?? to have a label next to the icon ?

website: :link:

Reply:
I was thinking the other way round: [:link: website](http://google.com)

Reply:
It's not so universal a symbol for website. I'm not sure a random user would know what it meant instantly. Maybe if the link icon was just to the right of the place name and the name itself was also a link, but as is I'm not sure it's clear enough.

Reply:
ahh okay this looks good now 
so what we gonna have is --->    Place: Haifa :link: [website](suckmyass)

Reply:
Or even **event name** :link:

Reply:
ok wow now we have 3 options choose one :laughing: 

1) Place: Haifa :link: [website](suckmyasss)

2) **event name** :link: 

3) not sure how it was in the start